etna/images/migrations/0002_migrate_old_image_data_to_new_model.py

```diff
@@ -80,8 +80,9 @@ class Migration(migrations.Migration):
         # Copy data from existing tables
         migrations.RunPython(migrate_forwards, migrations.RunPython.noop),
         # When running forwards, increment autoid to reflect additions
-        migrations.RunSQL(
-            'SELECT setval(pg_get_serial_sequence(\'"images_customimage"\',\'id\'), coalesce(max("id"), 1), max("id") IS NOT null) FROM "images_customimage";',
-            migrations.RunSQL.noop,
-        ),
+        # TODO: Figure out why this breaks...
+        # migrations.RunSQL(
+        #     'SELECT setval(pg_get_serial_sequence(\'"images_customimage"\',\'id\'), coalesce(max("id"), 1), max("id") IS NOT null) FROM "images_customimage";',
+        #     migrations.RunSQL.noop,
+        # ),
```

Comment:
TODO

Reply:
@ababic - I think you committed this migration. Do you know why it might break when I run `poetry run python manage.py migrate` on a completely fresh Postgres instance?

Reply:
@ahosgood I couldn't say for sure, but maybe it has something to do with there being no rows in the table. Now these migrations have been run everywhere that previously had data to migrate, it should be safe to lave this commented out